src/service/bookInstance.service.ts

```diff
@@ -0,0 +1,10 @@
+import { AppDataSource } from '../data-source';
+import { BookInstance } from './../entity/BookInstance';
+
+class BookInstanceService {
+    private BookInstanceRepository = AppDataSource.getRepository(BookInstance);
+    async countBookInstance() {
```

Comment:
Thêm line trống ở giữa để clear code hơn, tương tự cho các service khác nha
```suggestion
    private BookInstanceRepository = AppDataSource.getRepository(BookInstance);
    
    async countBookInstance() {
```